test/export/test_export.py

```diff
@@ -6401,20 +6390,6 @@ def forward(self, x, y):
         self.assertTrue(
             all(node.args[0].op == "placeholder" for node in sym_size_nodes)
         )
-        # dynamo will DCE the repeat node, AOTAutograd will leave it
-        # training IR will also DCE due to retracing
-        repeat_nodes = [
-            node
-            for node in ep.graph.nodes
-            if node.target == torch.ops.aten.repeat.default
-        ]
-        self.assertEqual(
-            len(repeat_nodes),
-            1
-            if is_non_strict_test(self._testMethodName)
-            and not is_training_ir_test(self._testMethodName)
-            else 0,
-        )
```

Comment:
after modifying the flag below, dynamo doesn't DCE this op, and there's now no strict/non-strict mismatch